### PR TITLE
feat: index GNPValue changed

### PR DESCRIPTION
**Pull request by m00n620**
Handle GNPValue changed event to index daily GNP values.

### CODE REVIEW

File: subgraph/schema.graphql [hunk 1]
```diff
@@ -33,6 +33,7 @@ type UnlockDailyData @entity {
   lockDeployed: BigInt!
   keysSold: BigInt!
   activeLocks: [Bytes!]
+  gnpValue: BigInt!
```
Comment (julien51) [inline subgraph/schema.graphql:36]:
```suggestion
  grossNetworkProduct: BigInt!
```

File: subgraph/src/unlock.ts [hunk 2]
```diff
@@ -39,6 +39,7 @@ export function handleNewLock(event: NewLock): void {
     unlockDailyData = new UnlockDailyData(dayID.toString())
     unlockDailyData.lockDeployed = BigInt.fromI32(1)
     unlockDailyData.keysSold = BigInt.fromI32(0)
+    unlockDailyData.gnpValue = BigInt.fromI32(0)
```
Comment (julien51) [inline subgraph/src/unlock.ts:42]:
```suggestion
    unlockDailyData.grossNetworkProduct = BigInt.fromI32(0)
```

File: subgraph/src/unlock.ts [hunk 3]
```diff
@@ -118,3 +119,13 @@ export function handleLockUpgraded(event: LockUpgraded): void {
     lock.save()
   }
 }
+
+export function handleGNPChanged(event: GNPChanged): void {
+  let timestamp = event.block.timestamp.toI32()
+  let dayID = timestamp / 86400
+  let unlockDailyData = UnlockDailyData.load(dayID.toString())
+  if (unlockDailyData) {
+    unlockDailyData.gnpValue = event.params.grossNetworkProduct
```
Comment (julien51) [inline subgraph/src/unlock.ts:128]:
```suggestion
    unlockDailyData.grossNetworkProduct = event.params.grossNetworkProduct
```

File: subgraph/tests/lockdaydata.test.ts [hunk 4]
```diff
@@ -43,5 +43,6 @@ describe('Describe UnlockDailyData Events', () => {
       'activeLocks',
       `[]`
     )
+    assert.fieldEquals('UnlockDailyData', lockDayID.toString(), 'gnpValue', '0')
```
Comment (julien51) [inline subgraph/tests/lockdaydata.test.ts:46]:
```suggestion
    assert.fieldEquals('UnlockDailyData', lockDayID.toString(), 'grossNetworkProduct', '0')
```

**Review by julien51 — APPROVED** [review]

**Comment by wonderwomancode** [comment]
@m00n620 I'd love to nominate you for retroactive funding for your contributions. I just need a wallet address.

**Comment by m00n620** [comment]
> @m00n620 I'd love to nominate you for retroactive funding for your contributions. I just need a wallet address.

0xA56A762cC7b19f7e7188677F06679B4532f64878, thanks @wonderwomancode

**Pull request merged**